data.sql

```diff
@@ -3,4 +3,12 @@ INSERT INTO animals (name, date_of_birth,escape_attempts, neutered,weight_kg) VA
  ('Gabumon', '11-15-2018', 2, 'True', 8),
   ('Pikachu', '01-07-2021', 1, 'False', 15.04),
   ('Devimon', '05-12-2017', 5, 'True', 11);
-  
\ No newline at end of file
+
```

Comment:
- [ ] You are not using the latest version of readme file given by microverse. 😨 But no problem 😊 you can have it from this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍

Reply:
Modified everything following the review